{
  "uid" : "cc2e5b16d95dc617",
  "name" : "Verify presence, visibility and structure of content on the page",
  "fullName" : "tests.contributors_page_test.TestContributorsPageStructure#test_cnp_01_01_verify_page_presence_and_visibility",
  "historyId" : "8c88035ce350231b478cea9e5f0a51bc",
  "time" : {
    "start" : 1728784114129,
    "stop" : 1728784114129,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55619f29b02a <unknown>\n#1 0x55619ef815e0 <unknown>\n#2 0x55619ef797eb <unknown>\n#3 0x55619ef69a09 <unknown>\n#4 0x55619ef6b744 <unknown>\n#5 0x55619ef69cbd <unknown>\n#6 0x55619ef6952f <unknown>\n#7 0x55619ef69422 <unknown>\n#8 0x55619ef67493 <unknown>\n#9 0x55619ef67ada <unknown>\n#10 0x55619ef83f41 <unknown>\n#11 0x55619f0158c5 <unknown>\n#12 0x55619eff5b22 <unknown>\n#13 0x55619f014d7d <unknown>\n#14 0x55619eff58c3 <unknown>\n#15 0x55619efc36b3 <unknown>\n#16 0x55619efc468e <unknown>\n#17 0x55619f265a2b <unknown>\n#18 0x55619f2699b1 <unknown>\n#19 0x55619f252225 <unknown>\n#20 0x55619f26a532 <unknown>\n#21 0x55619f23738f <unknown>\n#22 0x55619f289f28 <unknown>\n#23 0x55619f28a0f3 <unknown>\n#24 0x55619f299e7c <unknown>\n#25 0x7fd9fba94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"6a7be437803a26e979dc13938a39dd0f\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_CONTRIBUTORS_PAGE}')\n    def contributors_page_open(driver):\n>       driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n\ntests/conftest.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f747350>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55619f28a0f3 \\\\u003Cunknown>\\\\n#24 0x55619f299e7c \\\\u003Cunknown>\\\\n#25 0x7fd9fba94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55619f29b02a <unknown>\nE       #1 0x55619ef815e0 <unknown>\nE       #2 0x55619ef797eb <unknown>\nE       #3 0x55619ef69a09 <unknown>\nE       #4 0x55619ef6b744 <unknown>\nE       #5 0x55619ef69cbd <unknown>\nE       #6 0x55619ef6952f <unknown>\nE       #7 0x55619ef69422 <unknown>\nE       #8 0x55619ef67493 <unknown>\nE       #9 0x55619ef67ada <unknown>\nE       #10 0x55619ef83f41 <unknown>\nE       #11 0x55619f0158c5 <unknown>\nE       #12 0x55619eff5b22 <unknown>\nE       #13 0x55619f014d7d <unknown>\nE       #14 0x55619eff58c3 <unknown>\nE       #15 0x55619efc36b3 <unknown>\nE       #16 0x55619efc468e <unknown>\nE       #17 0x55619f265a2b <unknown>\nE       #18 0x55619f2699b1 <unknown>\nE       #19 0x55619f252225 <unknown>\nE       #20 0x55619f26a532 <unknown>\nE       #21 0x55619f23738f <unknown>\nE       #22 0x55619f289f28 <unknown>\nE       #23 0x55619f28a0f3 <unknown>\nE       #24 0x55619f299e7c <unknown>\nE       #25 0x7fd9fba94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728784114130,
      "stop" : 1728784114902,
      "duration" : 772
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "contributors_page_open",
    "time" : {
      "start" : 1728784114902,
      "stop" : 1728784115333,
      "duration" : 431
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55619f29b02a <unknown>\n#1 0x55619ef815e0 <unknown>\n#2 0x55619ef797eb <unknown>\n#3 0x55619ef69a09 <unknown>\n#4 0x55619ef6b744 <unknown>\n#5 0x55619ef69cbd <unknown>\n#6 0x55619ef6952f <unknown>\n#7 0x55619ef69422 <unknown>\n#8 0x55619ef67493 <unknown>\n#9 0x55619ef67ada <unknown>\n#10 0x55619ef83f41 <unknown>\n#11 0x55619f0158c5 <unknown>\n#12 0x55619eff5b22 <unknown>\n#13 0x55619f014d7d <unknown>\n#14 0x55619eff58c3 <unknown>\n#15 0x55619efc36b3 <unknown>\n#16 0x55619efc468e <unknown>\n#17 0x55619f265a2b <unknown>\n#18 0x55619f2699b1 <unknown>\n#19 0x55619f252225 <unknown>\n#20 0x55619f26a532 <unknown>\n#21 0x55619f23738f <unknown>\n#22 0x55619f289f28 <unknown>\n#23 0x55619f28a0f3 <unknown>\n#24 0x55619f299e7c <unknown>\n#25 0x7fd9fba94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 28, in contributors_page_open\n    driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/contributors",
      "time" : {
        "start" : 1728784114902,
        "stop" : 1728784115333,
        "duration" : 431
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55619f29b02a <unknown>\n#1 0x55619ef815e0 <unknown>\n#2 0x55619ef797eb <unknown>\n#3 0x55619ef69a09 <unknown>\n#4 0x55619ef6b744 <unknown>\n#5 0x55619ef69cbd <unknown>\n#6 0x55619ef6952f <unknown>\n#7 0x55619ef69422 <unknown>\n#8 0x55619ef67493 <unknown>\n#9 0x55619ef67ada <unknown>\n#10 0x55619ef83f41 <unknown>\n#11 0x55619f0158c5 <unknown>\n#12 0x55619eff5b22 <unknown>\n#13 0x55619f014d7d <unknown>\n#14 0x55619eff58c3 <unknown>\n#15 0x55619efc36b3 <unknown>\n#16 0x55619efc468e <unknown>\n#17 0x55619f265a2b <unknown>\n#18 0x55619f2699b1 <unknown>\n#19 0x55619f252225 <unknown>\n#20 0x55619f26a532 <unknown>\n#21 0x55619f23738f <unknown>\n#22 0x55619f289f28 <unknown>\n#23 0x55619f28a0f3 <unknown>\n#24 0x55619f299e7c <unknown>\n#25 0x7fd9fba94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 28, in contributors_page_open\n    driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"6a7be437803a26e979dc13938a39dd0f\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784115364,
      "stop" : 1728784115426,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Contributors Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "contributors_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestContributorsPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.contributors_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 58,
        "unknown" : 0,
        "total" : 61
      },
      "items" : [ {
        "uid" : "f92446e034e067d7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/f92446e034e067d7",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55826ea2e02a <unknown>\n#1 0x55826e7145e0 <unknown>\n#2 0x55826e70c7eb <unknown>\n#3 0x55826e6fca09 <unknown>\n#4 0x55826e6fe744 <unknown>\n#5 0x55826e6fccbd <unknown>\n#6 0x55826e6fc52f <unknown>\n#7 0x55826e6fc422 <unknown>\n#8 0x55826e6fa493 <unknown>\n#9 0x55826e6faada <unknown>\n#10 0x55826e716f41 <unknown>\n#11 0x55826e7a88c5 <unknown>\n#12 0x55826e788b22 <unknown>\n#13 0x55826e7a7d7d <unknown>\n#14 0x55826e7888c3 <unknown>\n#15 0x55826e7566b3 <unknown>\n#16 0x55826e75768e <unknown>\n#17 0x55826e9f8a2b <unknown>\n#18 0x55826e9fc9b1 <unknown>\n#19 0x55826e9e5225 <unknown>\n#20 0x55826e9fd532 <unknown>\n#21 0x55826e9ca38f <unknown>\n#22 0x55826ea1cf28 <unknown>\n#23 0x55826ea1d0f3 <unknown>\n#24 0x55826ea2ce7c <unknown>\n#25 0x7f16c8094ac3 <unknown>",
        "time" : {
          "start" : 1728697228096,
          "stop" : 1728697228096,
          "duration" : 0
        }
      }, {
        "uid" : "761c73d96a1e8f00",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/761c73d96a1e8f00",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x560a7129402a <unknown>\n#1 0x560a70f7a5e0 <unknown>\n#2 0x560a70f727eb <unknown>\n#3 0x560a70f62a09 <unknown>\n#4 0x560a70f64744 <unknown>\n#5 0x560a70f62cbd <unknown>\n#6 0x560a70f6252f <unknown>\n#7 0x560a70f62422 <unknown>\n#8 0x560a70f60493 <unknown>\n#9 0x560a70f60ada <unknown>\n#10 0x560a70f7cf41 <unknown>\n#11 0x560a7100e8c5 <unknown>\n#12 0x560a70feeb22 <unknown>\n#13 0x560a7100dd7d <unknown>\n#14 0x560a70fee8c3 <unknown>\n#15 0x560a70fbc6b3 <unknown>\n#16 0x560a70fbd68e <unknown>\n#17 0x560a7125ea2b <unknown>\n#18 0x560a712629b1 <unknown>\n#19 0x560a7124b225 <unknown>\n#20 0x560a71263532 <unknown>\n#21 0x560a7123038f <unknown>\n#22 0x560a71282f28 <unknown>\n#23 0x560a712830f3 <unknown>\n#24 0x560a71292e7c <unknown>\n#25 0x7ffa14e94ac3 <unknown>",
        "time" : {
          "start" : 1728610919440,
          "stop" : 1728610919440,
          "duration" : 0
        }
      }, {
        "uid" : "c702cd9918a95130",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/c702cd9918a95130",
        "status" : "passed",
        "time" : {
          "start" : 1728524594865,
          "stop" : 1728524594906,
          "duration" : 41
        }
      }, {
        "uid" : "8e52d47f87852d4a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/8e52d47f87852d4a",
        "status" : "passed",
        "time" : {
          "start" : 1728438166811,
          "stop" : 1728438166855,
          "duration" : 44
        }
      }, {
        "uid" : "4614889130c9ec86",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/4614889130c9ec86",
        "status" : "passed",
        "time" : {
          "start" : 1728351825557,
          "stop" : 1728351825596,
          "duration" : 39
        }
      }, {
        "uid" : "b7333dd1d9258b05",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/b7333dd1d9258b05",
        "status" : "passed",
        "time" : {
          "start" : 1728265612574,
          "stop" : 1728265613639,
          "duration" : 1065
        }
      }, {
        "uid" : "570905aed9387592",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/570905aed9387592",
        "status" : "passed",
        "time" : {
          "start" : 1728179383264,
          "stop" : 1728179383307,
          "duration" : 43
        }
      }, {
        "uid" : "f61543ddec55411c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/f61543ddec55411c",
        "status" : "passed",
        "time" : {
          "start" : 1728092548345,
          "stop" : 1728092548382,
          "duration" : 37
        }
      }, {
        "uid" : "4389cf70c862a180",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/4389cf70c862a180",
        "status" : "passed",
        "time" : {
          "start" : 1728006254185,
          "stop" : 1728006254268,
          "duration" : 83
        }
      }, {
        "uid" : "73dd375b5b6eaa34",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/73dd375b5b6eaa34",
        "status" : "passed",
        "time" : {
          "start" : 1727919841984,
          "stop" : 1727919842029,
          "duration" : 45
        }
      }, {
        "uid" : "5291cd6426a97e56",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/5291cd6426a97e56",
        "status" : "passed",
        "time" : {
          "start" : 1727833391347,
          "stop" : 1727833391384,
          "duration" : 37
        }
      }, {
        "uid" : "69631cc4610fc271",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/69631cc4610fc271",
        "status" : "passed",
        "time" : {
          "start" : 1727747527623,
          "stop" : 1727747527669,
          "duration" : 46
        }
      }, {
        "uid" : "83464e63e12c690a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/83464e63e12c690a",
        "status" : "passed",
        "time" : {
          "start" : 1727660808840,
          "stop" : 1727660808882,
          "duration" : 42
        }
      }, {
        "uid" : "596fda1dde6e4976",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/596fda1dde6e4976",
        "status" : "passed",
        "time" : {
          "start" : 1727574612903,
          "stop" : 1727574612943,
          "duration" : 40
        }
      }, {
        "uid" : "f1dced2b754dcd19",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/f1dced2b754dcd19",
        "status" : "passed",
        "time" : {
          "start" : 1727487863220,
          "stop" : 1727487863258,
          "duration" : 38
        }
      }, {
        "uid" : "7b7cf4af6e801a19",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/7b7cf4af6e801a19",
        "status" : "passed",
        "time" : {
          "start" : 1727401436319,
          "stop" : 1727401436361,
          "duration" : 42
        }
      }, {
        "uid" : "71218e85fbd5ca29",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/71218e85fbd5ca29",
        "status" : "passed",
        "time" : {
          "start" : 1727314982984,
          "stop" : 1727314983027,
          "duration" : 43
        }
      }, {
        "uid" : "26b859963c939c13",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/26b859963c939c13",
        "status" : "passed",
        "time" : {
          "start" : 1727228673625,
          "stop" : 1727228673667,
          "duration" : 42
        }
      }, {
        "uid" : "c136102158ed38b4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/c136102158ed38b4",
        "status" : "passed",
        "time" : {
          "start" : 1727142235026,
          "stop" : 1727142235109,
          "duration" : 83
        }
      }, {
        "uid" : "7a934f2992fe7b28",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/7a934f2992fe7b28",
        "status" : "passed",
        "time" : {
          "start" : 1727055789234,
          "stop" : 1727055789268,
          "duration" : 34
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "cc2e5b16d95dc617.json",
  "parameterValues" : [ ]
}